uwrt_mars_rover_utils/uwrt_mars_rover_xbox_controller/CMakeLists.txt

```diff
@@ -0,0 +1,122 @@
+cmake_minimum_required(VERSION 3.5)
+project(uwrt_mars_rover_xbox_controller)
```

Comment:
This is no longer true, should be changed to the correct package name. Right now we have it as uwrt_mars_rover_utils... which I dislike because even Colin has a utilites package. We should change this package to uwrt_mars_rover_comms. What do you think @keyonjerome ?